sdk/core/azure-core/test/ut/file_upload.cpp

```diff
@@ -11,9 +11,10 @@
 #include <Windows.h>
 #endif // Windows
 
+#include "azure/core/http/http.hpp"
+
 #include "transport_adapter.hpp"
-//#include <iostream>
-#include <response.hpp>
```

Comment:
Was this include unnecessary before?

Reply:
Good catch, I cleaned out a commented include and added back the wrong header.